shell/test/specs/demo-tests.js

```diff
@@ -437,7 +442,9 @@ describe('Arcs demos', function() {
     let restaurantNodes = pierceShadows(restaurantSelectors);
     console.log(`click: restaurantSelectors`);
     browser.elementIdClick(restaurantNodes.value[0].ELEMENT);
+    waitForStillnessAndOpenSystemUi();
```

Comment:
I'm not sure why this is needed. When I run the test in debug mode, I can see the suggestion show up, but the test fails with `unknown error: Element is not clickable at point (578, 871)` error, as in the suggestion is found, but not accepted.